src/features/ModelPlan/ModelToOperations/_components/Modal/SolutionForm/index.tsx

```diff
@@ -262,8 +263,19 @@ const SolutionForm = ({ closeModal }: { closeModal: () => void }) => {
           <Trans
             i18nKey={t('modal.solution.alert.info')}
             components={{
-              s: <span className="text-underline text-primary-light" />
-              // TODO: Add a link to the documentation
+              s: (
+                <Button
+                  type="button"
+                  className="usa-button usa-button--unstyled margin-top-0"
+                  onClick={() => {
+                    history.push(
+                      `/models/${modelID}/collaboration-area/model-to-operations/solution-library`
+                    );
```

Comment:
Solution Library url does not exist, but just inferring based on the milestone library url

Reply:
I think that makes sense!